client/my-sites/stats/stats-purchase/stats-purchase-tier-upgrade-slider.tsx

```diff
@@ -0,0 +1,73 @@
+import { PricingSlider } from '@automattic/components';
+import classNames from 'classnames';
+import { translate } from 'i18n-calypso';
+import React, { useState } from 'react';
+import useAvailableUpgradeTiers from './stats-purchase-tier-upgrade-slider-utils';
+
+import './stats-purchase-tier-upgrade-slider.scss';
+
+function getLocalizedStrings() {
+	const limits = translate( 'Monthly site views limit', {
+		comment: 'Heading for Stats Upgrade slider. The monthly views limit.',
+	} );
+	const price = translate( 'You will pay', {
+		comment: 'Heading for Stats Upgrade slider. The monthly price.',
+	} );
+	const strategy = translate( 'Price per month, billed yearly', {
+		comment: 'Stats Upgrade slider message. The billing strategy.',
+	} );
+
+	return {
+		limits,
+		price,
+		strategy,
+	};
+}
+
+type TierUpgradeSliderProps = {
```

Comment:
Let's place the type at the top of the file or in a dedicated file.

Reply:
Added this on to the task list as well.